sourcetracker/_sourcetracker.py

```diff
@@ -961,6 +961,11 @@ def _gibbs(source_df, sink_df, alpha1, alpha2, beta, restarts,
     >>> mp, mps = _gibbs(source_df, sink_df, alpha1, alpha2, beta, restarts,
                          draws_per_restart, burnin, delay, cluster=c)
     '''
+    if source_df.isnull().any().any() or sink_df.isnull().any().any():
```

Comment:
`df.isnull().values.sum()` will be a bit faster. Probably a bit prematurely optimizing but it it's an easy change